frontends/main/src/app-pages/DashboardPage/CoursewareDisplay/EnrollmentDisplay.test.tsx

```diff
@@ -75,21 +76,15 @@ describe("EnrollmentDisplay", () => {
 
     screen.getByRole("heading", { name: "My Learning" })
 
-    const desktopCards = await screen.findAllByTestId("enrollment-card-desktop")
-    const mobileCards = await screen.findAllByTestId("enrollment-card-mobile")
-    expect(desktopCards.length).toBe(7)
-    expect(mobileCards.length).toBe(7)
+    const cards = await screen.findAllByTestId("enrollment-card-desktop")
```

Comment:
Removed the desktop / mobile distinction, since that seems like an implementation detail of DashboardCard to me.